docs/components/Chips/Web.stories.tsx

```diff
@@ -91,9 +84,11 @@ const SelectionTemplate: ComponentStory<typeof Chips> = args => {
     handleSelect,
     handleCustomAdd,
   } = useFakeOptionQuery();
+
   return (
     <Chips
       {...args}
+      type="dismissible"
```

Comment:
I had issues with XOR in this branch. I have a feeling a recent update to ts-xor caused issues with the original implementation of the prop splitting for single/multiple/select. 

I updated the XOR usage to the typescript unary operator and made the usage of the different chipp types explicit and there's no longer errors about the activator being undefined like in the x-or implementation (which it should have been able to be for the single-select elsewhere in this test file)